app/models/user.rb

```diff
@@ -316,9 +316,15 @@ def liked_pages
   end
 
   def send_digest_email
-    top_picks = content_followed_in_period(1.week.ago, Time.now)
-    if top_picks.count.positive?
-      SubscriptionMailer.send_digest(id, top_picks).deliver_now
+    if self.has_tag('digest:daily')
+      top_picks = content_followed_in_period(Time.now - 1.day, Time.now)
```

Comment:
Not sure, but I think `@siaw23` was able to use `1.day.ago` instead of subtracting... maybe worth a try cause it's short?